pages/test_analytics/javascript_collectors.md.erb

```diff
@@ -53,3 +53,12 @@ Before you start, make sure Jest runs with access to [CI environment variables](
 Once you're done, in your Test Analytics dashboard, you'll see analytics of test executions on all branches that include this code.
 
 If you don't see branch names, build numbers, or commit hashes in the Test Analytics UI, then see [CI Environments](/docs/test-analytics/ci-environments) to learn more about exporting your environment to Jest.
+
+## Troubleshooting missing test executions and --forceExit
+
+Using the [--forceExit](https://jestjs.io/docs/cli#--forceexit) option when running Jest could result in missing test executions from Test Analytics.
+
+--forceExit could potentially terminate any ongoing processes that are attempting to send test executions to Buildkite.
```

Comment:
Shouldn't --forceExit be formatted as `--forceExit` @ L61?..

Reply:
Yep, I've updated to used backticks on this line at the two links. If the links look good, I'll hit merge.